jib-core/src/test/java/com/google/cloud/tools/jib/builder/steps/PullBaseImageStepTest.java

```diff
@@ -62,8 +67,10 @@
 public class PullBaseImageStepTest {
```

Comment:
should there be a test for all registry mirrors failing and last resort is to use the original registry url?